lib/Runtime/Base/ScriptContext.h

```diff
@@ -52,6 +52,8 @@ class SRCINFO
     Js::ModuleID moduleID;
     ULONG grfsi;
 
+    Js::Var serializedScriptBuffer; // in order to keep a reference to serialized Script Buffer
```

Comment:
@jianchun  That can be ifdef'ed for ChakraFull. I just don't want to update all the way down to JsrtSourceHolder. Eventually one thing is part of another.